backpack/extensions/firstorder/batch_grad/batchnorm_nd.py

```diff
@@ -0,0 +1,18 @@
+"""Contains grad_batch extension for BatchNorm."""
+from backpack.core.derivatives.batchnorm_nd import BatchNormNdDerivatives
+from backpack.extensions.firstorder.batch_grad.batch_grad_base import BatchGradBase
+from backpack.utils.errors import batch_norm_raise_error_if_train
+
+
+class BatchGradBatchNormNd(BatchGradBase):
+    """BatchGrad extension for BatchNorm."""
+
+    def __init__(self):
+        """Initialization."""
+        super().__init__(
+            derivatives=BatchNormNdDerivatives(), params=["bias", "weight"]
+        )
+
+    def apply(self, ext, module, g_inp, g_out):  # noqa: D102
+        batch_norm_raise_error_if_train(module, raise_error=False)
```

Comment:
for `batch_grad` there is a test:
`test/automated_test.py::test_batch_gradients_sum_to_grad`
Therefore, I only warn instead of raising an error.